index.html

```diff
@@ -1897,7 +1941,7 @@
                   <dfn><code>onconnected</code></dfn>
                 </td>
                 <td>
-                  <code>connected</code>
+                  <dfn><code>connected</code></dfn>
```

Comment:
As said in previous line note, the enum definitions are attached to a particular namespace (or prefix), so these definitions do not conflict with them.

Now, we could also add a namespace for these definitions to avoid linking to them by mistake when we rather mean to link to the enum. That would force us to specify a namespace each time we want to reference the term "connected". That is up to you :)